Lib/argparse.py

```diff
@@ -1064,6 +1064,7 @@ def __init__(self,
                  prog,
                  parser_class,
                  dest=SUPPRESS,
+                 required=True,
                  help=None,
                  metavar=None):
```

Comment:
For maximum backward compatibility, the new parameter should be added to the end.

(I don’t personnally write calls like `something('abc', SUPPRESS, True, None, None)` but they are supported, and should not be broken when possible.)

Reply:
`add_subparsers` itself takes no keyword arguments so I don't think there's a concern of the private api here.

```python
>>> parser.add_subparsers('foo', True, False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: add_subparsers() takes 1 positional argument but 4 were given
```

(the 1 positional argument here being `self`)

The ordering I chose was intentional to match the signatures in the rest of the file.

Reply:
Perfect!

Reply:
@merwok do I monty-python it up now?  I'm unclear on the process for dismissing a review :P

Reply:
The review holding this up is «please update docs and news» :)

Reply:
whoops totally missed that! I'll get right on it :)